src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -156,6 +165,19 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
 
     bool equal (Step const& rhs) const override;
 
+    // Iterate through the offers at the best quality in a book.
+    // Unfunded offers and bad offers are skipped (and returned).
+    // callback is called with the offer SLE, taker pays, taker gets.
+    // If callback returns false, don't process any more offers.
+    // Return the unfunded and bad offers and the number of offers consumed.
```

Comment:
This is a copy of the comment on the definition. Remove the one on the definition.